ui/src/components/ui/switch.tsx

```diff
@@ -24,13 +24,15 @@ const Switch = React.forwardRef<
     >
         <SwitchPrimitives.Thumb
             className={cn(
-                `pointer-events-none block h-5 w-5 rounded-full bg-white shadow-lg ring-0
+                `pointer-events-none block rounded-full bg-white shadow-lg ring-0
          transition-transform data-[state=checked]:translate-x-5 data-[state=unchecked]:translate-x-0
-          dark:bg-slate-950`
+          dark:bg-slate-950`,
+                thumbSize
```

Comment:
Could you rename this thumbClassName so that it is clearer that you can set any className for Thumb, not just to change size? Thanks.